vnclauncher.py

```diff
@@ -128,9 +129,28 @@ def connectVNC(self, button):
         cmd = '/usr/bin/x11vnc'
         if os.path.isfile(cmd) and os.access(cmd, os.X_OK):
             logging.debug('Using x11vnc installed in the system')
+            self._vte.feed_child(cmd.encode('utf-8'))
         else:
-            logging.error('Specified package : x11vnc does not found')
-        self._vte.feed_child(cmd.encode('utf-8'))
+            dialog=Gtk.MessageDialog(
+            transient_for=self,
+            flags=0,
+            message_type=Gtk.MessageType.INFO,
+            buttons=Gtk.ButtonsType.OK_CANCEL,
+            text="x11vnc is not installed"
+            )
+            dialog.format_secondary_text(
+                "Install x11vnc by clicking on OK"
+            )
+            response = dialog.run()
+
+            if response == Gtk.ResponseType.OK:
+                logging.debug("Installing x11vnc")
+                if platform.version().find("Ubuntu") > -1 or platform.version().find("Debian"):
+                    cmd = "apt install x11vnc"
+                if platform.version().find("Fedora") > -1:
```

Comment:
Might be better to use `platform.linux_distribution()[0]` because on fedora systems, `platform.version().find("Fedora")` will always return `-1` as `Fedora` isn't part of the string returned by `platform.version()`.

Reply:
Find a way to check for `Fedora` distros.

Reply:
`platform.platform()` contains the necessary information.

Reply:
Thanks Done in c733b9b.